docs/maintainers/maintainers-guide.md

```diff
@@ -43,66 +43,72 @@ There are two ways to add repositories to your Repository Insight Page:
 
 There are three tabs that provide you with more information on these repositories:
 
-#### Reports
+#### Repositories Dashboard
 
-The Reports tab allows paid users to filter for more information about their repositories over the last 30 days and to generate a Downloadable CSV. Filters include: Top 100 Repos, Minimum 5 Contributors, Recent, Most Active, and Spammed.
+This tab allows you to view more detailed information on each repository, including activity levels, PR overview, PR velocity, spam PRs, contributors, and activity stats over the last 30 days.
 
-#### Repositories
-
-The Repositories tab allows you to view more detailed information on each repository, including Activity, PR Overview, PR Velocity, Spam, Contributors, and activity over the last 30 days. To learn more about these features, see [Understanding Repository Insights Data](understanding-repo-insights.md).
+To learn and understand more about the data provided, see [Understanding Repository Insights Data](understanding-repo-insights.md).
 
 ![repositories tab](../../static/img/repos-insights.png)
 
 #### Contributors
 
-The Contributors tab allows you to view more detailed information on each contributor, including Activity, Repositories, the date of their last contribution, Time Zone, and the number of contributions.
+The "Contributors" tab allows you to view more detailed information on each contributor, including activity levels, the number of repositories they contributed to, the date of their last contribution, their most used language, their time zone, the number of contributions, and their activity stats over the last 30 days.
+
+You can select and add your contributors to a [Contributor Insight Page](#contributor-insights-connecting-with-contributors) to learn more about them.
 
-To learn more about your contributors, you can select them and add them to a Contributor Insight Page.
+There are two ways to add selected contributors to a Contributor Insight Page:
+
+1. Create a new page.
+2. Add them to an existing page.
 
 ![contributors insight list](../../static/img/contributors-insights.png)
 
-There are two ways to add contributors to a Contributor Insight Page:
+#### Activity
+
+The "Activity" tab shows a graph of the last time each contributor created their PR and how many lines of code they've touched.
 
-1. Create a new Contributor Insight Page with selected contributors.
-2. Add selected contributors to an existing Contributor Insight Page.
+It also provides more detailed information on each contributor when you hover over their image, including their latest PRs and repositories they've contributed to. Clicking their image will bring you to their profile on OpenSauced. You can use the information in this tab to help you learn about their engagement and general interests. It will be helpful if you want to collaborate with them or consider recruiting a maintainer for your project.
```

Comment:
This is the "Activity" tab subsection that I added for the Maintainers Guide. Same wording, different purpose that's shown in the last two sentences here.

Reply:
Sounds good.